src/ui/components/TestSuite/views/TestMetadata/TestItemTree.tsx

```diff
@@ -57,28 +57,29 @@ function TestTree({
 }) {
   const testTree = useMemo(() => createTestTree(testItems), [testItems]);
 
-  return <TreeRenderer selectTestItem={selectTestItem} testTree={testTree} />;
+  return <TreeRenderer isTopScope={true} selectTestItem={selectTestItem} testTree={testTree} />;
 }
 
 function TreeRenderer({
-  isNested = false,
+  isTopScope = false,
   scope = null,
   selectTestItem,
   testTree,
 }: {
-  isNested?: boolean;
+  isTopScope?: boolean;
   scope?: string | null;
   selectTestItem: (testItem: ProcessedTestItem) => void;
   testTree: TestTree;
 }) {
-  const parentScope = scope;
+  const className = scope ? styles.TreeNested : styles.Tree;
+  const scopeClassName = isTopScope ? styles.TopScope : styles.Scope;
```

Comment:
What's the purpose of this change?

There is no `TopScope` class in the CSS module for this component, so this looks like a mistake.

Reply:
Yup, I was going down this road in order to style the parent differently from nested items, but changed my mind. Then I forgot to remove this code, so this was indeed a mistake. Thanks!